src/pkg/notifier/model/event.go

```diff
@@ -78,3 +80,27 @@ type ReplicationResource struct {
 	Provider     string `json:"provider,omitempty"`
 	Namespace    string `json:"namespace,omitempty"`
 }
+
+// Retention describes tag retention infos
```

Comment:
the pkg/notifier should be a common package, it should not contain any specific model. 

We can define them into src/controller/event/model. OR use the `Custom` attribute of EventData.

BTW, please remove the replication model as well.

Reply:
moved to src/controller/event/model